src/services/orgUploadToken/useRegenerateOrgUploadToken.ts

```diff
@@ -1,24 +1,41 @@
 import { useMutation, useQueryClient } from '@tanstack/react-query'
 import { useParams } from 'react-router-dom'
+import { z } from 'zod'
 
 import Api from 'shared/api'
+import { NetworkErrorObject } from 'shared/api/helpers'
+
+const RequestSchema = z.object({
+  regenerateOrgUploadToken: z
+    .object({
+      orgUploadToken: z.string().nullish(),
+      error: z
+        .union([
```

Comment:
Can we swap this over to a `. discriminatedUnion`, this will result in faster parsing times, and a bit better language server performance, because it's only checking against on field of the objects, instead of each field etc.

Reply:
done